spec/integration/apo_spec.rb

```diff
@@ -48,7 +58,12 @@
     expect(page).to have_selector('.permissionName', text: 'someone')
     expect(find_field('desc_md').value).to eq('MODS')
     expect(find_field('use_license').value).to eq('by-sa')
-    expect(page).to have_link('New Testing Collection')
+    # TODO: This was removed when we switched from using
```

Comment:
If we split this test into a "create" test and an "edit" test, could we restore this assertion?

Reply:
Plausible, for sure. I'll give it a shot.

Reply:
Not happy with the attempt I made. Still too tangly. Inclined to pass on it for now, and revisit when dor-services-app is dockerized.